src/js/api.js

```diff
@@ -54,27 +55,27 @@ export function getLatestSubmission(){
 }
 
 export function getEditsByType(submission){
-  return sendFetch(`/submissions/${submission}/edits`)
+  return sendFetch('/submissions/${submission}/edits')
```

Comment:
These should be the original grave marks because we're using ES2015 string substitution here